api/authenticators/okta/okta_auth.go

```diff
@@ -0,0 +1,301 @@
+package okta
```

Comment:
This file handles working with the `/api/v1/authn` API from Okta. This isn't provided to us by the SDK so we have to do it ourselves.